projects/extension/ai/vectorizer.py

```diff
@@ -28,7 +28,29 @@ def execute_vectorizer(plpy, vectorizer_id: int) -> None:
         plpy.error(f"vectorizer {vectorizer_id} not found")
 
     vectorizer = json.loads(result[0]["vectorizer"])
+    _execute_vectorizer(plpy, vectorizer)
 
+def execute_vectorizer_by_name(plpy, vectorizer_name: str) -> None:
+    try:
+        plan = plpy.prepare("""
+            select pg_catalog.to_jsonb(v) as vectorizer
+            from ai.vectorizer v
+            where name = $1
+            limit 1
+        """, ["text"])
+        result = plan.execute([vectorizer_name], 1)
+    except Exception as e:
+        if 'column "name" does not exist' in str(e):
```

Comment:
An alternative to this would be to check for the column via `information_schema.columns` table, but that will mean we would be doing one extra query always, even in the case where `name` column is presence (last versions of pgai lib)